nft_ingester/src/bin/backfill/main.rs

```diff
@@ -306,6 +310,9 @@ async fn main() {
         let _ = handle.await;
     }
 
+    if stop_handle.await.is_err() {
```

Comment:
Can you explain what are we awaiting for here?

Reply:
for all the tasks we spawned using `usecase::executor::spawn` to join, making the shutdown graceful